Source/utils/stdcompat/string_view_split.hpp

```diff
@@ -0,0 +1,92 @@
+#include "utils/stdcompat/string_view.hpp"
```

Comment:
This is not a polyfill for a standard library functions, so it shouldn't be in stdcompat.
Please move to `util/string_split_by_char.hpp`